charts/generic-prometheus-alerts/templates/rules.yaml

```diff
@@ -0,0 +1,31 @@
+{{- $targetApplication := required "A value for targetApplication must be set" .Values.targetApplication }}
+---
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: {{ $targetApplication }}-rules
+  labels:
+    {{- include "generic-prometheus-alerts.labels" . | nindent 4 }}
+spec:
+  groups:
+    - name: {{ $targetApplication }}-rules
+      rules:
+        - record: summer_time_offset
+          expr: |
+            (vector(1) and (month() > 3 and month() < 10))
+            or
+            (vector(1) and (month() == 3 and (day_of_month() - day_of_week()) >= 25) and absent((day_of_month() >= 25) and (day_of_week() == 0)))
+            or
+            (vector(1) and (month() == 10 and (day_of_month() - day_of_week()) < 25) and absent((day_of_month() >= 25) and (day_of_week() == 0)))
+            or
+            (vector(1) and ((month() == 10 and hour() < 1) or (month() == 3 and hour() > 0)) and ((day_of_month() >= 25) and (day_of_week() == 0)))
+            or
+            vector(0)
+        - record: british_time
+          expr: time() + 3600 * summer_time_offset
+        - record: hmpps_business_day
+          expr: 0 < day_of_week(british_time) < 6
```

Comment:
would great to make these  configurable values to the chart (with this as default), so teams can set their own working hours.  We could add this later though.